pages/windows/clip.md

```diff
@@ -5,8 +5,16 @@
 
 - Pipe command line output to the Windows clipboard:
 
-`{{dir}} | clip`
+`dir|clip`
```

Comment:
Could this change be reverted please? The spacing helps to aid clarity in the command, and the `{{` `}}` token syntax is actually a tldr-specific extension to markdown that indicates that a user can change a value. It isn't rendered in the final page that's shown to the user, as I'm sure you've noticed.